project-base/templates/Front/Inline/Cart/cartBox.html.twig

```diff
@@ -27,3 +61,29 @@
         </a>
     </div>
 </div>
+
+{% macro cartItem(cartItem) %}
+    {% set product = cartItem.product %}
+    <li class="cart-detail__item">
+        <div class="cart-detail__item__image">
+            {{ image(product) }}
+        </div>
+        <div class="cart-detail__item__content">
+            <a class="cart-detail__item__content__title" href="{{ url('front_product_detail', {'id': product.id}) }}" title="{{ product.name }}">
+                {{ product.name }}
+            </a>
+            <span class="cart-detail__item__content__price">
+                {{ cartItem.watchedPrice|price}}
+            </span>
+        </div>
+        <div class="cart-detail__item__pieces">
+            {{ cartItem.quantity|formatNumber }} {{ product.unit.name }}
+        </div>
+        <a
+                class="cart-detail__item__remove js-cart-item-remove-button"
```

Comment:
I see js class `js-cart-item-remove-button` but I dont see some javascript who use this class (same as `js-cart-box-content` on line 27). if I understand this code - we'll show the cart items after hover and when a customer wants to remove an item, customer will be redirected to the cart?  Don't we want to create some ajax request?

Reply:
Yes, we discussed it it with @DavidOstrozlik
I have same opinion - we should remove item from cart hover content via Ajax and reload content.

Reply:
Ok, can I prepare BE route for it? Or how did the discussion turn out? :smile:

Reply:
I don´t think I can handle it. So it is your turn :)

Reply:
Ok

Reply:
Fixed